### PR TITLE
aggregator: fix available condition transition time

### PR DESCRIPTION
**Pull request by sttts**
Fixes regression introduced in https://github.com/kubernetes/kubernetes/pull/71380.

### CODE REVIEW

**Review by logicalhan — APPROVED** [review]
/lgtm

File: staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go [hunk 1]
```diff
@@ -151,8 +152,9 @@ func (c *AvailableConditionController) sync(key string) error {
 	apiService := originalAPIService.DeepCopy()
 
 	availableCondition := apiregistration.APIServiceCondition{
-		Type:   apiregistration.Available,
-		Status: apiregistration.ConditionTrue,
+		Type:               apiregistration.Available,
+		Status:             apiregistration.ConditionTrue,
+		LastTransitionTime: metav1.Now(),
```
Comment (liggitt) [inline staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go]: can we add a test that would have caught the issue?

Comment (sttts) [reply staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go]: done

**Pull request merged**